src/lib/components/UI/Header.svelte

```
@@ -1,92 +1,239 @@
+      <div class="flex items-center gap-2">
+        <svg class="w-4 h-4 text-green-200" fill="currentColor" viewBox="0 0 20 20">
+          <path fill-rule="evenodd" d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7-4a1 1 0 11-2 0 1 1 0 012 0zM9 9a1 1 0 000 2v3a1 1 0 001 1h1a1 1 0 100-2v-3a1 1 0 00-1-1H9z" clip-rule="evenodd"/>
+        </svg>
+        <span class="font-medium">New: Enhanced LoRaWAN connectivity now available!</span>
+        <a href="#learn-more" class="underline hover:no-underline ml-2 hover:translate-x-2 transition-all ease-in-out">Learn more →</a>
+      </div>
+      <button onclick={closeAnnouncement} class="text-green-200 hover:text-white transition-colors">
+        <svg class="w-4 h-4" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+          <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12"/>
+        </svg>
+      </button>
```

ESLint

[error] Buttons and links should either contain text or have an `aria-label` or `aria-labelledby` attribute
https://svelte.dev/e/a11y_consider_explicit_label(a11y_consider_explicit_label)

(svelte/valid-compile)